### DISCUSSION TITLE
No audio in downloads from MotoGP.com Videopass

### DISCUSSION THREAD

**Discussion by Trnopr** [aclap-dev/video-downloadhelper#1530] [Help and Q&A]
### Your question

As per title:
downloads using Chrome on the MotoGP.com videopass website have no audio (audio plays fine on the website itself) . If I try the "download audio" option in Video Downloadhelper I get the DOWNLOAD FAILED message.
Any fixes or tips?
(my first post so please be kind if I have made any etiquette errors)

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"f4be852514504375"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 2,
    "successfull_dl": 89,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You'll want to study #625.

**Reply by Trnopr** [reply]
Thanks for your reply.
I'm not going to lie, most of that information went so far over my head that even a telescope won't help; I gather the gist of it is it needs a fix. 
At least I know it isn't something I am doing wrong.
Many thanks once again.
